backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VitualQueueController.java

```diff
@@ -36,4 +42,9 @@
     public CreateShopResponse addShop(@RequestBody CreateShopRequest createShopRequest) {
         return virtualQueueService.createShop(createShopRequest);
     }
+
+    @GetMapping("/token/{shopId}")
+    public List<Token> getTokens(@PathVariable UUID shopId) {
+		return virtualQueueService.getTokens(shopId);
```

Comment:
I think this return statement should be in line (wrt indentation) with the previous return statement (on line 43). Please check once.

Reply:
this is showing on my local fine! please verify by cloning repo!